pl-jogurt-z-orzechami-wloskimi-i-miodem.md

```diff
@@ -0,0 +1,29 @@
+# Jogurt z orzechami włoskimi i miodem
+
+## Składniki
+
+### Deser
+
+-   Pełnotłusty jogurt Bieluch/grecki 150g,
+-   Orzechy włoskie 25-30g,
+-   Miód (najlepiej tymiankowy lub inny jasny kwiatowy) 20g,
+
+
+## Przygotowanie
+
+### Deser
+
+1. Jogurt mieszamy na gładko w misce.
+2. Orzechy kruszymy lub siekamy - dodajemy na wierz jogurtu.
+3. Na orzechy wylewamy miód.
+4. Przed spożyciem mieszamy.
+
+## Uwagi
+
+Orzechy włoskie można zastąpić dowolnymi innymi orzechami - też dobre.
+
+Owoce (jagody, wiśnie czy co tam w sezonie) jako dodatek są git ale wtedy mniej miodu.
+
+## Ocena smaku _pizza & pasta_
+
+**9+1/10**
```

Comment:
Czm 9+1? 😮

Reply:
bo `10` to `9 + 1` - podkreślam `+1`, spoko napiszę normalnie